### DISCUSSION TITLE
Estimate hours needed to complete

### DISCUSSION THREAD

**Discussion by Jdkol3s** [Alooishus/ci_recipe_book#7] [General]
What is a road map or list of everything you would need to complete and launch?

**Comment by Alooishus** [comment]
I created milestones just to group issues as I make them.  If I had to guess, fast and ugly completion time from where I am at, it is about 8 hours/1 day.  With all the nitpicking and making it pretty, I would estimate around 40 hours maybe more of coding.

**Comment by Jdkol3s** [comment]
Sounds good... Let me know if I can be involved. Probably a lot of learning on my part

**Comment by Alooishus** [comment]
I think that would be awesome and I would love for anyone interested to get involved.  
I’ll try and keep the live site up to date with this code base so people can mess around and request features or notify of bugs and whatnot.

**Comment by Alooishus** [comment]
I can easily talk you through a local environment setup as well if you want the code to mess around with!
